knplabs/dictionary-bundle/2.0/config/packages/knp_dictionary.yaml

```diff
@@ -0,0 +1,17 @@
+knp_dictionary:
+    dictionaries: []
+
+# Examples
+# knp_dictionary:
```

Comment:
I'm -1 for this commented example. Files copied into the project are not meant to be documentation. And this commented section is not something they can uncomment, as it would make the YAML file invalid (duplicate key)

Reply:
@stof :+1: fixed